.github/workflows/actions.yml

```diff
@@ -3,13 +3,17 @@ name: JUnit Pioneer Site
 on:
   push:
     branches: [ grandmaster, site-source ]
+  repository_dispatch:
+    types: [triggerSiteBuild]
 
 jobs:
   publish:
     runs-on: ubuntu-latest
     steps:
       - name: Check out repo
         uses: actions/checkout@v2
+        with:
+          ref: "grandmaster"
```

Comment:
i think this action should only run on grandmaster, or am i wrong here, also regarding the build scripts

Reply:
> grandmaster: contains all branches (as worktrees) and projects (as clones or symlinks)

So yes, this correct I guess